vars/createTestJobPipeline.groovy

```diff
@@ -0,0 +1,89 @@
+#!groovy
+
+def call() {
+
+    def builder = getJenkinsLabels("aws", "eu-west-1")
+
+    pipeline {
+        agent {
+            label {
+                   label builder.label
+            }
+        }
+        environment {
+            AWS_ACCESS_KEY_ID     = credentials('qa-aws-secret-key-id')
+            AWS_SECRET_ACCESS_KEY = credentials('qa-aws-secret-access-key')
+        }
+        parameters {
+            string(defaultValue: "",
+               description: 'folder name or path in jenkins jobs structure',
+               name: 'branch')
+            string(defaultValue: "master",
+               description: 'sct branch',
+               name: 'sct_branch')
+            string(defaultValue: "git@github.com:scylladb/scylla-cluster-tests.git",
+               description: 'sct repo link',
+               name: 'sct_repo')
+            booleanParam(defaultValue: false,
+                description: "Create Enterprise test job",
+                name: 'is_enterprise')
+        }
+        options {
+            timestamps()
+            disableConcurrentBuilds()
+            buildDiscarder(logRotator(numToKeepStr: '20'))
+        }
+        stages {
+            stage('Checkout sct') {
+                steps {
+                    script {
+                        completed_stages = [:]
+                    }
+                    dir('scylla-cluster-tests') {
+                        timeout(time: 5, unit: 'MINUTES') {
+                            checkout scm
+
+                            dir("scylla-qa-internal") {
```

Comment:
We don't need this repo to create Jenkins jobs, it's used in perf regression user-profiles and longevity 5000 tables tests....

Reply:
fixed